File: Persistence/Models/Project.cs
using System.ComponentModel.DataAnnotations;

namespace Persistence
{
    public class Project
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }

        // Navigation properties
        public List<int> AssignmentIds { get; set; } // Collection navigation
    }
}

File: teamo_webreactapp/src/components/project/Project.js
import React from 'react'

function Project(id, name, description, startDate, endDate) {
  return (
    <div>
        <h5>{id}</h5>
        <h5>{name}</h5>
        <h5>{description}</h5>
        <h5>{startDate}</h5>
        <h5>{endDate}</h5>
    </div>
  )
}

export default Project